src/main/resources/org/jenkinsci/plugins/workflow/cps/CpsScmFlowDefinition/config.jelly

```diff
@@ -29,6 +29,9 @@ THE SOFTWARE.
     <f:entry field="scriptPath" title="${%Script Path}">
         <f:textbox default="Jenkinsfile"/>
     </f:entry>
+    <f:entry field="lightweight" title="${%Lightweight checkout}">
```

Comment:
This box is selected by default. I think it is safer to maintain existing behavior for existing Pipelines.